src/jsonGraphToCypher.js

```diff
@@ -106,14 +103,14 @@ const jsonGraphToCypher = async (graph, sourceGraphId) => {
       if (isLocation) {
         rps.push(`${nodeName}.point = point({latitude:toFloat(${props.latitude}), longitude:toFloat(${props.longitude})})`);
       }
-      typedProperties = `${rps.join(', ')}, `;
+      typedProperties = `,  ${rps.join(', ')}`.trim();
```

Comment:
It seems like this change would cause a syntax error for the cypher query e.g. `...SET , name = 'foo' ...`